instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/decorator/BaseDecorator.java

```diff
@@ -17,10 +17,10 @@
 package io.opentelemetry.instrumentation.api.decorator;
 
 import static io.opentelemetry.OpenTelemetry.getPropagators;
+import static io.opentelemetry.instrumentation.api.tracer.utils.NetPeerUtils.ENDPOINT_PEER_SERVICE_MAPPING;
```

Comment:
Looks like more than `Utils` now

Reply:
Yes - but on the other hand, once we remove all `BaseDecorator` extensions this constant will go back to being private.